server/scraper/resto/menu_manual.py

```diff
@@ -638,6 +638,17 @@ def teambuilding_2023_en(_path, original):
     original["open"] = False
     return original
 
+def newyear_dsv_2024_nl(_path, original):
+    add_or_append(original, "Op vrijdag 12 januari is resto De Brug gesloten, cafetaria De Brug is wel open. ")
+    original["open"] = True
```

Comment:
I've put open to True to make sure that people will still see the menu for the other locations.